content/_data/upgrades/2-303-1.adoc

```diff
@@ -0,0 +1,176 @@
+* `jenkins/agent:latest-jdk8`
+* `jenkins/agent:latest-archlinux-jdk8`
+* `jenkins/agent:bullseye-jdk8`
+* `jenkins/agent:4.10-1-jdk8`
+* `jenkins/agent:4.10-1-alpine-jdk8`
+* `jenkins/agent:4.10-1-windowsservercore-ltsc2019-jdk8`
+
+==== Obsolete Docker images and tags no longer published
+
+Jenkins Docker images for new releases are not published for `centos`.
+The `centos` images use CentOS 8 as a baseline and CentOS 8 is no longer receiving updates from the Red Hat Enterprise Linux upstream.
+
+Jenkins Docker Windows images for new releases are not published for the `1809` images.
+Microsoft has ended mainstream support for `1809`.
+
+Jenkins Docker tags that include a version number are no longer published with the redundant `-lts` tag.
+The transition looks like this:
+
+* 2.303.1-lts is replaced with 2.303.1
+* 2.303.1-lts-alpine is replaced with 2.303.1-alpine
+* 2.303.1-lts-centos7 is replaced with 2.303.1-centos7
+* 2.303.1-lts-slim is replaced with 2.303.1-slim
+
+==== Digester 2 removed
+
+The Apache Commons Digester v2.1 has been removed from Jenkins core.
+The library was last released in 2010.
+Plugins that depend on the library need to be updated to remove that dependency.
+
+Plugins that are affected include:
+
+* link:https://plugins.jenkins.io/vs-code-metrics/[Visual Studio Code Metrics]
+* link:https://plugins.jenkins.io/BlameSubversion/[Blame Subversion]
+* link:https://plugins.jenkins.io/javatest-report/[JavaTest report]
+* link:https://plugins.jenkins.io/vss/[Visual SourceSafe]
+* link:https://plugins.jenkins.io/synergy/[Synergy]
+* link:https://plugins.jenkins.io/config-rotator/[Config Rotator]
+* link:https://plugins.jenkins.io/harvest/[Harvest SCM]
+* link:https://plugins.jenkins.io/cmvc/[CMVC]
+
+Suspended plugins (no longer distributed by the Jenkins update center) that are affected include:
+
+* link:https://plugins.jenkins.io/tfs/[TFS]
+* link:https://issues.jenkins-ci.org/browse/INFRA-2487[svn-release-mgr]
+* link:https://plugins.jenkins.io/cpptest/[cpptest]
+* link:https://plugins.jenkins.io/CFLint/[CFLint]
+
+==== Stop bundling External Monitor Job Type, LDAP, and PAM plugins with Jenkins
+
+Certain key Jenkins capabilities were initially created inside the Jenkins core.
+As Jenkins has developed further, capabilities moved from inside Jenkins core to dedicated Jenkins plugins, like the plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], and the plugin:pam[PAM authentication plugin].
+When those plugins were created, the plugins were "bundled" inside the jenkins.war file to retain compatibility for plugins that depended on their functionality to be inside Jenkins core.
+The plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], and the plugin:pam[PAM authentication plugin] are no longer bundled with Jenkins.
+
+In very rare cases, this could result in problems when attempting to install plugins compatible with Jenkins (then Hudson) before 1.467.
```

Comment:
```suggestion
In very rare cases, this could result in problems when attempting to install plugins compatible with Jenkins before 1.467.
```

1.467 was already Jenkins. I suggest removing Hudson reference at all